include/ElemDataRequests.h

```diff
@@ -138,6 +138,16 @@ class ElemDataRequests
     add_face_field(*meta_.get_fields()[field], scalarsPerFace);
   }
 
+  inline void add_ip_field(unsigned field, unsigned scalarsPerNode)
```

Comment:
All other kernels (see `ScalarOpenAdvElemKernel`) used `add_face_field`, can we not reuse that method to be consistent with the rest of the code.